pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
+, enableZlib ? false, zlib
+, enablePng ? false, libpng
+, enableEigen ? false, eigen }:
+
+stdenv.mkDerivation rec {
+  version = "1.1";
+  name = "openmvg-${version}";
+
+  src = fetchgit {
+    url = "https://www.github.com/openmvg/openmvg.git";
+
+    # Tag v1.1
+    rev = "f5ecb48";
+    sha256 = "1di9i7yxnkdvl8lhflynmqw62gaxwv00r1sd7nzzs9qn63g0af0f";
+    fetchSubmodules = true;
+  };
+
+  buildInputs =
+       [ glibc ]
```

Comment:
Providing a libc is the job of `stdenv`; explicit passing of `glibc` is likely to make porting the package to `darwin` (macOS) platform harder (unless just removing `glibc` is enough, of course…).